lib/cli/commands/subscribeorders.ts

```diff
@@ -15,20 +15,51 @@ export const builder = {
 };
 
 export const handler = (argv: Arguments) => {
+  ensureConnection(argv, true);
+};
+
+const ensureConnection = (argv: Arguments, printError?: boolean) => {
+  loadXudClient(argv).getInfo(new xudrpc.GetInfoRequest(), (error: Error | null) => {
```

Comment:
perhaps you can use `loadXudClient(argv).waitForReady` instead

Reply:
Will check that